packages/store-sdk/src/analytics/events.ts

```diff
@@ -0,0 +1,185 @@
+export interface PageViewData {
```

Comment:
For this interface, I'm not sure if we should keep the `accountName` or the `pageType` handles. Also, why not renaming pageUrl to url,  pageTitle to title and so on?

Reply:
I'll rename all of these events, including the `vtex:` references in another PR. I think it will be clearer, making it easier to review. Renaming all those events in this PR and restructuring them to follow GA4's structure would turn it into an enormous PR